src/compute-engine/boxed-expression/boxed-function.ts

```diff
@@ -681,7 +681,7 @@ export class BoxedFunction extends _BoxedExpression {
     }
 
     // root(sqrt(a), c) -> root(a, 2*c)
-    if (this.operator === 'Sqrt' || this.operator === 'Root') {
+    if (this.operator === 'Sqrt') {
```

Comment:
(^WRT commit message)
I did remove `!BoxedExpr.value now altogether returns 'undefined' for non-literal
  boxed-expr. types (e.g. functions)`, (an older edit) from the message, but this has somehow persisted.

Reply:
OK, not sure what happened...